pcdet/datasets/dataset.py

```diff
@@ -130,6 +131,30 @@ def __getitem__(self, index):
         """
         raise NotImplementedError
 
+    def set_lidar_aug_matrix(self, data_dict):
```

Comment:
I think it will be better to move this function to `data_augmentor.py` and call it after forward all data augmentation.

Reply:
The function `set_lidar_aug_matrix` needs to be called during testing and identity matrix is generated, so it may not be appropriate to move it `data_augmentor.py`.